pegasus/helpers/section_api_helpers.rb

```diff
@@ -113,9 +113,9 @@ def self.completed_levels(user_id)
       and("best_result >= #{ActivityConstants::MINIMUM_PASS_RESULT}")
   end
 
-  VALID_GENDERS = %w(m f)
+  @@valid_genders = %w(m f)
```

Comment:
All of these constants complained of redefinition due to the way these helpers are loaded in the tests.  Using class variables makes these not warn - though maybe we want to live with warnings in tests and get real warnings if we redefine these?